.github/workflows/integration_tests.yml

```diff
@@ -47,4 +47,10 @@ jobs:
         working-directory: integration_tests
         env:
           FAL_STATS_ENABLED: false
-        run: behave
+        run: |
+          if [[ '${{ matrix.dbt }}' =~ ^0.*$ ]]
+          then
+            behave --tags ~@dbtv1
```

Comment:
The way we had said before "exclude these" with a tag was 

```
behave --tags=-broken_profile
```

https://behave.readthedocs.io/en/stable/tutorial.html#controlling-things-with-tags
>then running behave --tags=slow will run just the scenarios tagged @slow. If you wish to check everything except the slow ones then you may run behave --tags=-slow.